style.css

```diff
@@ -133,8 +134,52 @@ body {
   }
 
   .menu {
-    display: none;
+    display: block;
     position: fixed;
+    top: 0;
+    left: -100%;
+    width: 100%;
+    height: 100%;
+    margin: 0 0 0 0;
+    padding: 177px 0 0 0;
+    list-style: none;
+    text-align: center;
+    background-color: #fff;
+    box-shadow: 1px 0 6px rgba(0, 0, 0, .2);
+    transition-duration: .25s;
+    z-index: 10;
+  }
+
+  .menu__item {
+    display: block;
+    padding: 12px 24px;
+    color: #333;
+    font: 500 30px/35px Roboto, arial, helvetica, sans-serif;
+    text-decoration: none;
+    transition-duration: .25s;
+  }
+
+  .menu__item_active::after {
+    display: none;
+  } 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/HTML_CSS_2.0-Artem1998/pullRequest?prid=6399509)